herddb-core/src/main/java/herddb/core/TableSpaceManager.java

```diff
@@ -1602,6 +1621,39 @@ public TableSpaceCheckpoint(
             this.tablesCheckpoints = tablesCheckpoints;
         }
     }
+    
+    public void createAndWriteTableDigest(TableSpaceManager manager, String tableSpace,String table ){
```

Comment:
what about returning a structure with: tablespacename, tablename, digest, number of records, next auto increment value (even tables without auto increment)